thread/src/components/RequestsSent.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
```

Comment:
This whole file is in the wrong location. Pulling from `go-live-branding` will fix that